packit/base_git.py

```diff
@@ -431,6 +431,15 @@ def download_remote_sources(self):
         the configuration match to the URL basename from SourceX) or from the one
         from SourceX in specfile.
         """
+        # Fetch all sources defined in packit.yaml -> sources
+        if self.package_config.sources:
```

Comment:
`self.package_config.sources` should be an empty list by default, so this can be removed